modules/pom.xml

```diff
@@ -0,0 +1,395 @@
+                                            <property name="jboss.folder" value="${JBOSS_FOLDER}" />
+                                            <target name="copy-server-dir"/>
+                                        </ant>
+                                    </target>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+
+            <modules>
+                <module>testcases/jdkAll/Eap</module>
+            </modules>
+            
+        </profile>
+
+        <profile>
+            <id>wildfly.testsuite.onserver</id>
+            <activation>
+                <file>
+                    <exists>../../wildfly-testsuite</exists>
+                </file>
+            </activation>
+
+            <properties>
+                <JBOSS_FOLDER>../../../dist/target/wildfly-7.0.0.ER5-redhat-SNAPSHOT/</JBOSS_FOLDER>
+            </properties>
+
+            <modules>
+                <module>testcases/jdkAll/Wildfly</module>
+            </modules>
+
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>prepare</id>
+                                <inherited>false</inherited>
+                                <phase>generate-resources</phase>
+                                <goals>
+                                    <goal>run</goal>
+                                </goals>
+                                <configuration>
+                                    <target>
+                                        <ant antfile="${basedir}/src/test/scripts/copy-server-dir.xml">
+                                            <property name="todir" value="${basedir}/wildfly/build/target/jbossas" />
+                                            <property name="jboss.folder" value="${JBOSS_FOLDER}" />
+                                            <target name="copy-server-dir"/>
+                                        </ant>
+                                    </target>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+
+        </profile>
+
+        <profile>
+            <id>wildfly.jdk8.testsuite.onserver</id>
+            <activation>
+                <file>
+                    <exists>../../wildfly-jdk8-testsuite</exists>
+                </file>
+            </activation>
+
+            <properties>
+                <JBOSS_FOLDER>../../../dist/target/wildfly-7.0.0.ER5-redhat-SNAPSHOT/</JBOSS_FOLDER>
```

Comment:
The other solution is to configure it as an env variable it teamcity project agent. But since the folder name depends on the version we would have to do it for every version. (Or we could change the name of the produced folder to something independent of the version and configure the env variable in teamcity just once.)